packages/common/src/modules/register.ts

```diff
@@ -0,0 +1,87 @@
+import ConfigService from '../services/ConfigService';
+import SettingsService from '../services/SettingsService';
+
+import WatchHistoryController from '../stores/WatchHistoryController';
+import CheckoutController from '../stores/CheckoutController';
+import AccountController from '../stores/AccountController';
+import ProfileController from '../stores/ProfileController';
+import FavoritesController from '../stores/FavoritesController';
+import AppController from '../stores/AppController';
+
+// Epg services
+import EpgService from '../services/EpgService';
+import ViewNexaEpgService from '../services/epg/ViewNexaEpgService';
+import JWEpgService from '../services/epg/JWEpgService';
+
+// Integration interfaces
+import AccountService from '../services/integrations/AccountService';
+import CheckoutService from '../services/integrations/CheckoutService';
+import SubscriptionService from '../services/integrations/SubscriptionService';
+import ProfileService from '../services/integrations/ProfileService';
+
+// Cleeng integration
+import CleengService from '../services/integrations/cleeng/CleengService';
+import CleengAccountService from '../services/integrations/cleeng/CleengAccountService';
+import CleengCheckoutService from '../services/integrations/cleeng/CleengCheckoutService';
+import CleengSubscriptionService from '../services/integrations/cleeng/CleengSubscriptionService';
+
+// InPlayer integration
+import JWPAccountService from '../services/integrations/jwp/JWPAccountService';
+import JWPCheckoutService from '../services/integrations/jwp/JWPCheckoutService';
+import JWPSubscriptionService from '../services/integrations/jwp/JWPSubscriptionService';
+import JWPProfileService from '../services/integrations/jwp/JWPProfileService';
+import { getIntegrationType } from './functions/getIntegrationType';
+import { isCleengIntegrationType, isJwpIntegrationType } from './functions/calculateIntegrationType';
+
+// Common services
+container.bind(ConfigService).toSelf();
+container.bind(EpgService).toSelf();
+container.bind(WatchHistoryService).toSelf();
+container.bind(FavoriteService).toSelf();
+container.bind(GenericEntitlementService).toSelf();
+container.bind(ApiService).toSelf();
+container.bind(SettingsService).toSelf();
+
+// Common controllers
+container.bind(AppController).toSelf();
+container.bind(WatchHistoryController).toSelf();
+container.bind(FavoritesController).toSelf();
+
+// Integration controllers
+container.bind(AccountController).toSelf();
+container.bind(CheckoutController).toSelf();
+container.bind(ProfileController).toSelf();
+
+// EPG services
+container.bind(EpgService).to(JWEpgService).whenTargetNamed(EPG_TYPE.jwp);
+container.bind(EpgService).to(ViewNexaEpgService).whenTargetNamed(EPG_TYPE.viewNexa);
+
+// Functions
+container.bind(INTEGRATION_TYPE).toDynamicValue(getIntegrationType);
+
+// Cleeng integration
+container.bind(DETERMINE_INTEGRATION_TYPE).toConstantValue(isCleengIntegrationType);
```

Comment:
Maybe something like `INTEGRATION_TYPE_CONFIGURATIONS` ?

Reply:
Doesn't that imply it has something to do with the integration configuration? We have coined `INTEGRATION_TYPE` for the type. This is the function that returns that type if it should use that integration. What do you think?

Reply:
Well, yeah. Also makes sense to me. @dbudzins any other suggestions?

Reply:
It's a minor change, we can update this later (but before the next release).